.github/workflows/pigz.yml

```
@@ -95,18 +93,18 @@ jobs:
         CI: true
 
     - name: Compile source code
-      run: cmake --build . -j2 --config ${{ matrix.build-config || 'Release' }}
+      run: cmake --build . -j5 --config ${{ matrix.build-config || 'Release' }}
```

actionlint (1.7.11)

[error] property "build-config" is not defined in object type {cmake-args: string; compiler: string; coverage: string; gcov-exec: string; name: string; os: string; packages: string}

(expression)